### PR TITLE
Update Daisy README

### PR DESCRIPTION
**Pull request by adjackura**
Add basic documentation and examples for workflow steps and the dependency map.

### CODE REVIEW

File: daisy/README.md [hunk 1]
```diff
@@ -1,4 +1,193 @@
-## Daisy
+# Daisy
 
 Daisy is a GCE workflow tool for running image type workflows on GCE. Development is ongoing.
+https://godoc.org/github.com/GoogleCloudPlatform/compute-image-tools/daisy
 
+Daisy takes a JSON workflow specification as input. This workflow contains the step information
+for the various steps of the workflow as well as a directed acyclic graph (DAG) of workflow steps.
+Each step may specify dependencies on other steps’ completion, but these dependencies must form a
```
Comment (zmarano) [inline daisy/README.md]: This is good information but details. The high level- this is what Daisy does should be spelled out in the simplest terms possible here.

File: daisy/README.md [hunk 2]
```diff
@@ -1,4 +1,193 @@
...
+    * [WaitForInstancesSignal](#waitforinstancessignal)
+    * [WaitForInstancesStopped](#waitforinstancesstopped)
+  * [Dependency Map](#dependency-map)
+
+## Workflow Steps
+A Daisy workflow consists of a set of steps and a dependancy map. Step types are defined here:
+https://godoc.org/github.com/GoogleCloudPlatform/compute-image-tools/daisy/workflow#Step
+
+In a workflow file the `steps` field is a mapping of step names to their type descriptions. The
+name can be whatever you chose, it's how you will reference the steps in the dependency map as
```
Comment (zmarano) [inline daisy/README.md]: choose

File: daisy/README.md [hunk 3]
```diff
@@ -1,4 +1,193 @@
-## Daisy
+# Daisy
 
 Daisy is a GCE workflow tool for running image type workflows on GCE. Development is ongoing.
```
Comment (zmarano) [inline daisy/README.md]: How about: "Daisy is a workflow tool for GCE images."

File: daisy/README.md [hunk 4]
```diff
@@ -1,4 +1,193 @@
...
+    * [WaitForInstancesStopped](#waitforinstancesstopped)
+  * [Dependency Map](#dependency-map)
+
+## Workflow Steps
+A Daisy workflow consists of a set of steps and a dependancy map. Step types are defined here:
+https://godoc.org/github.com/GoogleCloudPlatform/compute-image-tools/daisy/workflow#Step
+
+In a workflow file the `steps` field is a mapping of step names to their type descriptions. The
+name can be whatever you chose, it's how you will reference the steps in the dependency map as
+well as how they will show up in the logs. For each individual 'step' in steps you set one step
```
Comment (zmarano) [inline daisy/README.md]: TBD... but this wording needs help :)

File: daisy/README.md [hunk 5]
```diff
@@ -1,4 +1,193 @@
...
+}
+```
+
+### AttachDisks
+Not implemented yet.
+
+### CreateDisks
+Creates GCE disks.
+
+This CreateDisks step example creates two disks, the first is a standard PD disk created from a
```
Comment (zmarano) [inline daisy/README.md]: two disks. The First ...

File: daisy/README.md [hunk 6]
```diff
@@ -1,4 +1,193 @@
...
+      "sourceFile": "gs://my-bucket/image.tar.gz",
+      "noCleanup": true,
+      "exactName": true
+    }
+  ]
+}
+```
+
+### CreateInstances
+Creates GCE images.
```
Comment (zmarano) [inline daisy/README.md]: Creates GCE instances.

File: daisy/README.md [hunk 7]
```diff
@@ -1,4 +1,193 @@
...
+
+This CreateInstances step example  creates an instance with two attahced disks and uses
```
Comment (zmarano) [inline daisy/README.md]: Extra space...
"step example creates"...

Comment (zmarano) [inline daisy/README.md]: attached

File: daisy/README.md (hunk 1 above)
Comment (adjackura) [inline daisy/README.md]: dropped as it was unneeded info. We can figure out exactly how to describe daisy in the broad sense later.

**Pull request merged**